pkg/kubectl-argo-rollouts/info/analysisrun_info.go

```diff
@@ -26,32 +26,55 @@ func getAnalysisRunInfo(ownerUID types.UID, allAnalysisRuns []*v1alpha1.Analysis
 				UID:               run.UID,
 			},
 		}
+		if run.Spec.Metrics != nil {
+			arInfo.SuccessCondition = run.Spec.Metrics[0].SuccessCondition
+
+			if run.Spec.Metrics[0].Count != nil {
+				arInfo.Count = run.Spec.Metrics[0].Count.IntVal
+			}
+
+			if run.Spec.Metrics[0].InconclusiveLimit != nil {
+				arInfo.InconclusiveLimit = run.Spec.Metrics[0].InconclusiveLimit.IntVal
+			}
+			if run.Spec.Metrics[0].FailureLimit != nil {
+				arInfo.FailureLimit = run.Spec.Metrics[0].FailureLimit.IntVal
+			}
+		}
 		arInfo.Status = string(run.Status.Phase)
 		for _, mr := range run.Status.MetricResults {
 			arInfo.Successful += mr.Successful
 			arInfo.Failed += mr.Failed
 			arInfo.Inconclusive += mr.Inconclusive
 			arInfo.Error += mr.Error
-			lastMeasurement := analysisutil.LastMeasurement(run, mr.Name)
-			if lastMeasurement != nil && lastMeasurement.Metadata != nil {
-				if jobName, ok := lastMeasurement.Metadata[job.JobNameKey]; ok {
-					jobInfo := rollout.JobInfo{
-						ObjectMeta: &v1.ObjectMeta{
-							Name: jobName,
-						},
-						Icon:   analysisIcon(lastMeasurement.Phase),
-						Status: string(lastMeasurement.Phase),
+			for _, meas := range analysisutil.ArrayMeasurement(run, mr.Name) {
```

Comment:
instead of meas, you can call it lastMeasurement

Reply:
Its an array of measurements, we are iterating over each of the measurement. I can rename it to measurement instead of last measurement. It might give an idea that we are iterating over measurements. Please let me know your thoughts on this.

Reply:
sure, please rename as measurement